src/http/routes.js

```diff
@@ -63,6 +63,14 @@ router.get(
   hasVerifiedAccount,
   controllers.nurseUser.getSingleNurseUser,
 );
+router.put(
+  '/nurse/:nurseId',
+  authMiddleware,
+  hasUserPrivledge([HOSPITAL_ADMIN_USER.toLowerCase(), WARD_USER.toLowerCase()]),
+  hasConfirmedEmail,
+  hasVerifiedAccount,
```

Comment:
Since this endpoint is only accessible by a nurse user, we don't need the `hasConfirmedEmail` or `hasVerifiedAccount` here.

Reply:
fixed